### ISSUE TITLE
Bug/feature request: Mission npc from 'npc kill' missions regenerate their crew. 

### ISSUE THREAD

**Issue by shnwnd** [endless-sky/endless-sky#4872]
**Describe the bug**
Mission npc from 'npc kill' missions regenerate their crew. 

I am experiencing an issue where a mission npc from 'npc kill' regenerates its crew when I land on a planet and return to board with my crew refreshed.

**To Reproduce**
1. Get a bounty mission. (with a npc kill fleet)
2. Disable that bounty fleet.
3. Board a ship attempt to capture.
4. Attack without killing all the crew.
5. Defend without killing all the crew.
6. Count the crew.
7. Leave the boarding screen.
8. Leave the system.
9. Land on a planet.
10. Launch from planet.
11. Enter the system.
12. Board the same ship.
13. Count the crew. Are they the same as step 6?

**Expected behavior**
I expect the crew to stay at the number that remain when I left the boarding screen.
Where could they have gotten more crew?

- From a friendly ship? Why did that that friendly ship put more crew in danger? Why didn't that friendly ship repair/protect it? Why did that ship scarper off?
- From breeding/cloning/creating robots (at some point even that ought to be resource limited for humans right?)

**System (please complete the following information):**
 - OS: Chromebook 81.0.4044.31 (Official Build) beta (64-bit)
 - Game Source: self-compiled
 - Version: commit hash 330ebab7fe19e79e370652fae963004739cb5a3b

**Link to save file**
https://hastebin.com/jevuwolile
Enter Edusa to the north West. [B]oard either of the Korath Raiders there. Continue with step 4 To Reproduce.

**Is your feature request related to a problem? Please describe.**
I am frustrated when trying a tactic doesn't work with no worldbuilding reason.
This is a hunting tactic for humans, and is a behavior some predators exhibit.
Ware your opponent down over time, keep them in a position they cannot rest but you can.

**Describe the solution you'd like**
I would like to propose a few solutions that aren't exclusive of each other.

1. Have disabled ships sometimes get repaired. (edge cases with broken jump drive outfit/plundered hyperdrive) on own, or with help
2. Have disabled ships sometimes call for help to be protected.
3. Have disabled ships sometimes use up a resource from the cargo to create a random number of more crew (robots, biologicals, ?. Should we track the type of crew?)
4. Have disabled ships sometimes use up a resource from the cargo to create an amount of better defences. (traps, mines)
5. Have disabled ships sometimes use up a resource from the cargo to create other friendly ships? (consider making un/able to repair mothership?)
6. Have disabled ships sometimes self-destruct/get destroyed? (edge cases with some missions?)

Lots of ideas from boarding missions.

**Related Issue Links**
https://github.com/endless-sky/endless-sky/issues/1638
Now that we have a "never disabled" and "uncapturable" flags why continue to protect every NPC from this pirate/hunting tactic? (don't they self-destruct when never disabled or uncapturable or something?) Why not encourage the player to try interesting modes of play by fleshing out this action.

https://github.com/endless-sky/endless-sky/issues/2947
https://github.com/endless-sky/endless-sky/issues/2948
https://github.com/endless-sky/endless-sky/issues/871

**Comment by beccabunny** [comment]
The current behavior, which was added with the issue #1638 you mentioned, keeps boarding from becoming utterly overpowered again. Options from 1 to 5 would, imo, add lots of complexity in a place where it is not needed, and that would serve no purpose other than confusing the player. Option 6, instead, is used in some missions, but its behavior is much more tedious and limiting than the replenishing crew.
To sum it up: this is a case of gameplay > realism, in order to don't make boarding too easy and overpowered
